Source/Provers/SMTLib/SMTLibProverOptions.cs

```diff
@@ -30,7 +30,7 @@ public enum SolverKind
     Z3,
     CVC5,
     YICES2,
-    NoOp
+    NoOpWithZ3Options
```

Comment:
This is a bit odd.  Why the name change?

Reply:
To indicate that Z3 specific options should be set when using this solver.